test-tools/wamr-ide/VSCode-Extension/src/utilities/directoryUtilities.ts

```diff
@@ -118,3 +120,44 @@ export function checkFolderName(folderName: string) {
 
     return valid;
 }
+
+export function DownloadFile(url: string, destinationPath: string): Promise<void> {
```

Comment:
in general in TypeScript we use camelCase for function/methods (for example [here](https://google.github.io/styleguide/tsguide.html), [here](https://basarat.gitbook.io/typescript/styleguide#variable-and-function) and [here](https://google.github.io/styleguide/tsguide.html)) and PascalCase is used mainly for class names or interfaces. I'd agree keeping it as it is if that's consistent within the file; however, this file is a mixture of both (`checkFolderName`, `CheckIfDirectoryExist`) so I'd suggest following the standards accepted by industry; if possible, you might also rename the other functions but I understand that's outside of scope for this change.